### PR TITLE
Patch 3.0

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/GoToLine.cs [hunk 1]
```diff
@@ -85,10 +85,7 @@ private void TextBox1KeyDown(object sender, KeyEventArgs e)
                     e.KeyCode == Keys.End ||
                     (e.KeyValue >= 96 && e.KeyValue <= 105))
                 {
-                }
-                else if (e.KeyData == (Keys.Shift | Keys.Home) || e.KeyData == (Keys.Shift | Keys.End) ||
```
Comment (ivandrofly) [inline src/Forms/GoToLine.cs]: This code isn't needed 'cause it would never run... the first condition will cover it

Comment (niksedk) [inline src/Forms/GoToLine.cs]: Hm, looks good to me... could you explain a bit more?

Comment (ivandrofly) [inline src/Forms/GoToLine.cs]: https://github.com/ivandrofly/subtitleedit/blob/patch-3.0/src/Forms/GoToLine.cs#L70
Will cover the second condition which would capture the Shift+Home...

**Comment by niksedk** [comment]
Seems to work okay except the "e.Handled = true;" should not be there as it disables the the arrow/home/end keys (alternately replace it with "return" which might look better than empty brackets)

**Comment by ivandrofly** [comment]
@niksedk I'll close this pull and pull a new one which contain **return** instead

**Pull request closed without merging**